reference/provided-images.md

```diff
@@ -11,6 +11,8 @@ The following table lists supported images available on the official image serve
 
 | Name                        | Based on | Android Version | Ubuntu Version | Available since |
 |-----------------------------|----------|-----------------|----------------|---------------|
+| `jammy:android14:amd64`     | AOSP     | 14              | 22.04          | 1.24.0 |
+| `jammy:android14:arm64`     | AOSP     | 14              | 22.04          | 1.24.0 |
```

Comment:
```suggestion
| `jammy:aaos14:amd64`     | AAOS     | 14              | 22.04          | 1.24.0 |
| `jammy:aaos14:arm64`     | AAOS     | 14              | 22.04          | 1.24.0 |
| `jammy:android14:amd64`     | AOSP     | 14              | 22.04          | 1.24.0 |
| `jammy:android14:arm64`     | AOSP     | 14              | 22.04          | 1.24.0 |
```

Reply:
Oh, sorry, the [old PR description](https://github.com/canonical/anbox-cloud-docs/pull/190) that I wrote tripped me up. I added this but then removed it looking at the PR description. I will fix it.